towhee/operator/model_handler.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+
+
+class HandlerBase(ABC):
```

Comment:
The definition of `Handler` looks similar to `Operator`, is it possible to directly use `Operator` here?

Reply:
Or we should replacce `Operator` with `Handler`

Reply:
NNoperator is the entry point for both train and inference, handler is only for inference.